webapp/src/css/inbox.less

```diff
@@ -1292,6 +1299,7 @@ mm-sidebar-menu .mat-sidenav-container {
   mm-panel-header {
     background: @top-header-color;
     height: @sidebar-menu-header-desktop;
+    padding: 0px !important;
 
```

Comment:
Thanks for your work. As you mentioned i do these changes as suggested. But some i can't do because on my pc they cant override anything without !important :/.